gpg-interface.c

```diff
@@ -88,6 +88,9 @@ int check_signature(const char *payload, size_t plen, const char *signature,
 
 void print_signature_buffer(const struct signature_check *sigc, unsigned flags)
 {
+	if (flags & GPG_VERIFY_QUIET)
+		return;
```

Comment:
I think this is a very small and quite non-intrusive change. *Only* [`verify-commit.c`](https://github.com/lukpueh/git/blob/master/builtin/verify-commit.c#L29) and [`tag.c`](https://github.com/lukpueh/git/blob/master/tag.c#L27)  call the print function of the gpg-interface. And the addition of the flag does not even require an changes in those.

I think I presented a (your) good reason in the commit messages of these two commits.